Dockerfile

```diff
@@ -5,6 +5,8 @@
 
 FROM golang:latest
 
+RUN git config --global url."https://${GITHUB_OAUTH_TOKEN}@github.com/".insteadOf "git@github.com:"
```

Comment:
Before I merge the PR, I would like to know little bit more about the `GITHUB_OAUTH_TOKEN` variable. Is this something specific to CircleCI, or something from GitHub app? Can we expect it to be set for everybody?

This works with CircleCI, but I want to make sure the Dockerfile will work for all users.

Reply:
Whoops, sharp one! That's a leftover I will remove, copied this from a Go dockerfile where this token is needed to access private repositories.